port/raspberrypi/rp2xxx/src/hal/drivers.zig

```diff
@@ -323,3 +323,45 @@ pub const ClockDevice = struct {
         return @enumFromInt(t);
     }
 };
+
+const Cyw43PioSpi = microzig.hal.cyw49_pio_spi.Cyw43PioSpi;
+const Cyw43_Spi = microzig.drivers.wireless.Cyw43_Spi;
+const Cyw43_Bus = microzig.drivers.wireless.Cyw43_Bus;
+const Cyw43_Runner = microzig.drivers.wireless.Cyw43_Runner;
+
+pub const CYW43_Pio_Device_Config = struct {
+    spi: hal.cyw49_pio_spi.Cyw43PioSpi_Config,
+    pwr_pin: hal.gpio.Pin,
+};
+
+// TODO: CYW43 top level struct just for testing purpose (please redesign)
+pub const CYW43_Pio_Device = struct {
+    const Self = @This();
+    pwr_pin: GPIO_Device = undefined,
+    cyw43_pio_spi: Cyw43PioSpi = undefined,
+    cyw43_spi: Cyw43_Spi = undefined,
+    cyw43_bus: Cyw43_Bus = undefined,
+    cyw43_runner: Cyw43_Runner = undefined,
```

Comment:
If these must be set, we can remove the default value to make it a compile error to initialize it incorrectly.